### PR TITLE
Small Fixes

### PR DESCRIPTION
**Pull request by LegitDongo**
Fix some broken links
Fix some broken markdown
Small text edits

### CODE REVIEW

File: jobs/jobs.md [hunk 1]
```diff
@@ -8,19 +8,19 @@ In the last MAD version a job processor was implemented. With this tool are you
...
 
 ### Discord Alert
 
-see [Discord Alert](discord_alert)
+see [Discord Alert](discord_alert.md)
 
 ### MADmin Endpoint
 
-see [MADmin_Endpoint](MADmin_Endpoint)
+see [MADmin_Endpoint](MADmin_Endpoint.md)
```
Comment (muckelba) [inline jobs/jobs.md:23]: The links are working perfectly fine atm

**Review by muckelba — CHANGES_REQUESTED** [review]
The other fixes are fine, but probably needs a rebase or something after merging the rework from @sn0opy

**Comment by sn0opy** [comment]
Just pushed my rework directly to this repo as `rework` branch. Feel free to PR against it directly.

File: jobs/jobs.md (hunk 1 above)
Comment (LegitDongo) [reply jobs/jobs.md:23]: @muckelba see here for broken links
https://github.com/Map-A-Droid/MAD-docs/blob/master/jobs/jobs.md

I know this is made to be viewed on readthedocs, but some may find it on Github 🤷‍♂

**Pull request closed without merging**